resources/component_runit_supervisor_upstart.rb

```diff
@@ -0,0 +1,67 @@
+include ComponentRunitSupervisorResourceMixin
+
+provides :component_runit_supervisor, platform_family: 'rhel' do |node|
+  node['platform_version'].to_i == 6
+end
+provides :component_runit_supervisor, platform: 'fedora' do |node|
+  node['platform_version'].to_i <= 14
```

Comment:
i'm aggressively not giving a shit about those provides lines and just copying what was there, i think its all bad, but i don't have time to rethink it all right now.